### PR TITLE
add command to list all pillow checkpoint doc ids

### PR DESCRIPTION
**Pull request by czue**

### CODE REVIEW

File: pillowtop/management/commands/run_ptop.py [hunk 1]
```diff
@@ -20,6 +20,11 @@ class Command(NoArgsCommand):
                     dest='list_all',
                     default=False,
                     help="List pillowtop names"),
+        make_option('--list-checkpoints',
+                    action='store_true',
+                    dest='list_checkpoints',
+                    default=False,
+help="Print all pillow doc ids."),
```
Comment (millerdev) [inline pillowtop/management/commands/run_ptop.py]: funny indentation

File: pillowtop/management/commands/run_ptop.py [hunk 2]
```diff
@@ -31,11 +36,12 @@ class Command(NoArgsCommand):
...
-        run_all = options['run_all']
-        list_all = options['list_all']
-        pillow_name = options['pillow_name']
-        pillow_key = options['pillow_key']
+    def handle_noargs(self, run_all, list_all, list_checkpoints, pillow_name, pillow_key, **options):
+        run_all = run_all
+        list_all = list_all
+        list_checkpoints = list_checkpoints
+        pillow_name = pillow_name
+        pillow_key = pillow_key
```
Comment (millerdev) [inline pillowtop/management/commands/run_ptop.py]: Can these be removed? They appear to be no-ops

**Comment by czue** [comment]
sorry, this is screwy. closing till i can fix it

**Comment by czue** [comment]
@millerdev I think i must have accidentally pressed some sequence of keys that made my IDE do that automatic refactor on the function. fixed in https://github.com/dimagi/pillowtop/pull/106/

**Pull request closed without merging**